tests/common.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/portworx/torpedo/drivers"
 	"github.com/portworx/torpedo/drivers/backup"
 	"github.com/portworx/torpedo/pkg/osutils"
+
```

Comment:
Remove unnessassary empty lines.

Reply:
this is comming by default after saving the file